startup_scripts/000_users.py

```diff
@@ -5,19 +5,18 @@
 from users.models import Token
 
 users = load_yaml('/opt/netbox/initializers/users.yml')
-if users is None:
-  sys.exit()
+if not users is None:
```

Comment:
TBH, I'm not happy with the additional nesting this is causing. But I don't know another solution out of my head either :-/

Anyways, the following notion would be preferred by me and seems to be the preferred one [by the Python community at large](https://stackoverflow.com/a/2710949/172132):

```suggestion
if users is not None:
```